docs/Roadmap.md

```diff
@@ -47,48 +47,56 @@ ONNX Runtime already supports a wide range of architectures, platforms, and lang
 Additionally, we understand that lightweight devices and local applications may have constraints for package size, so there is active awareness to opportunistically minimize binary size.
 
 #### Architectures
-|Supported|Future|
-|---|---|
-|X64|Additional as-needed|
-|X86| |
-|ARM64 (Limited)| |
-|ARM32 (Limited)| |
+Supported
+* X64
+* X86
+* ARM64
+* ARM32 (Limited)
+
 
 #### Platforms
-|Supported|Future|
-|---|---|
-|Windows 7+|iOS|
-|Linux (various)| |
-|Mac OS X| |
-|Android (community contribution, Preview)| |
+Supported
+* Windows 7+
+* Linux (various)
+* Mac OS X
+* Android (community contribution, Preview)
+
+*Future*
+* *iOS*
 
 #### Languages
-|Supported|Future|
-|---|---|
-|Python (3.5, 3.6, 3.7)| Javascript |
-|C++| |
-|C#| |
-|C| |
-|Ruby (community project)| |
-|Java | |
+Supported
+* Python (3.5, 3.6, 3.7)
+* C++
+* C#
+* C
+* Ruby (community project)
+* Java
+* Javascript - Node.js (Preview)
 
 
 ### Accelerators and Execution Providers
 #### New EPs
 To achieve the best performance on a growing set of compute targets across cloud and the intelligent edge, we invest in and partner with hardware partners and community members to add new execution providers. The flexible pluggability of ONNX Runtime is critical to support a broad range of scenarios and compute options.
 
-|Supported|Future|
-|---|---|
-|MLAS (Microsoft Linear Algebra Subprograms)|AMD GPU|
-|Intel DNNL / MKL-ML|Xilinx FPGA|
-|Intel nGraph| |
-|NVIDIA CUDA| |
-|NVIDIA TensorRT| |
-|Intel OpenVINO| |
-|Nuphar Model Compiler| |
-|Microsoft Direct ML| |
-|Android NN API (Preview)| |
-|ARM Compute Library (Preview)| |
+Supported
+
+* MLAS (Microsoft Linear Algebra Subprograms)
+* Intel DNNL / MKL-ML
+* Intel nGraph
+* NVIDIA CUDA
+* NVIDIA TensorRT
+* Intel OpenVINO
+* Microsoft Direct ML
+* Nuphar Model Compiler (Preview)
+* Android NN API (Preview)
+* ARM Compute Library (Preview)
+* Rockchip NPU (Preview)
```

Comment:
I don't think I've plumbed the Rockchip NPU through to the Java API. Is it worth noting which EPs are supported by which APIs? I don't think the C# API supports all these either.

Reply:
This one is new and noted as Preview so functionality may be limited. The [Rockchip EP page](https://github.com/microsoft/onnxruntime/blob/master/docs/execution_providers/RKNPU-ExecutionProvider.md) should provide all the detailed notes for level of support while it's in preview. That said, since all the supported EPs are already listed in the new front-page readme, I may take this out of the roadmap and just list new ones there to reduce redundancy.